gulpfile.js

```diff
@@ -58,9 +57,10 @@ gulp.task('scripts', () => {
 // Runs tasks that copy assets to the public directory.
 // ---------------------------------------
 
-gulp.task('build', cb => {
-  runsequence('clean', ['styles', 'images', 'scripts'], cb)
-})
+gulp.task('build', gulp.series(
+  'clean',
+  ['styles', 'images', 'scripts']
```

Comment:
I can't find anything in gulp's documentation about passing arrays to `gulp.series` – what happens with these tasks? Are they run in series, or parallel?

If they run in series, do they need to be in an array or should this just be `gulp.series('clean', 'styles', 'images', 'scripts')`?

If they run in parallel, can we make that explicit by doing `gulp.series('clean', gulp.parallel('styles', 'images', 'scripts'))`?

(I may just be missing something here, `gulp` is not one of my strong points!)

Reply:
That's a good point, I've made it more explicit with `gulp.parallel`.